src/script/builder/tir/prim_func_frame.h

```diff
@@ -55,7 +59,25 @@ class PrimFuncFrame : public TIRFrame {
 
 PrimFuncFrame PrimFunc_(String name);
 tvm::tir::Var Arg(String name, tvm::tir::Var var);
-tvm::tir::Buffer Arg(String name, tvm::tir::Buffer buffer);
+tvm::tir::Var Arg(String name, tvm::tir::Buffer buffer);
+DictAttrs FuncAttrs(DictAttrs attrs);
```

Comment:
See my comment: https://github.com/junrushao1994/tvm/pull/42/files#r895044696

Also, there is probably no need to return anything

```suggestion
void FuncAttrs(Map<String, ObjectRef> attrs);
```